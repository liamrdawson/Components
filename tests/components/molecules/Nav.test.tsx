import { jsx } from '@emotion/react'
import {render} from '@testing-library/react'
import { Nav } from '../../../src'

test('Snapshot Test: Contact Form', () => {
  const pagesArray = [
    {
      pathName: 'path/to/page/one',
      pageName: 'Page One',
    },
    {
      pathName: 'path/to/page/two',
      pageName: 'Page Two',
    },
  ]

  const {container} = render(
    <Nav pages={pagesArray}>Heading Level One</Nav>,
  )

  expect(container).toMatchSnapshot()
})
