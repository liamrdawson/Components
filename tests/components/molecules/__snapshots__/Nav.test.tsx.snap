// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Snapshot Test: Contact Form 1`] = `
<div>
  <nav
    class="css-23iqwz-Nav"
  >
    <ul
      class="css-rk3qdn-Nav"
    >
      <li>
        <a
          class="css-r5wu0g"
          href="path/to/page/one"
        >
          <span
            class="css-r5wu0g"
          >
            <p
              class="css-8mhfhs-Text"
            >
              Page One
            </p>
          </span>
        </a>
      </li>
      <li>
        <a
          class="css-r5wu0g"
          href="path/to/page/two"
        >
          <span
            class="css-r5wu0g"
          >
            <p
              class="css-8mhfhs-Text"
            >
              Page Two
            </p>
          </span>
        </a>
      </li>
    </ul>
  </nav>
</div>
`;
